logger.js

```diff
@@ -304,9 +304,9 @@ logger.log_if_level = function (level, key, plugin) {
                 logger.log(
                     level,
                     [
-                        '[' + logobj.level + ']',
-                        '[' + logobj.uuid + ']',
-                        '[' + logobj.origin + ']',
+                        `[${logobj.level}]`,
+                        `[${logobj.uuid}]`,
+                        `[${logobj.origin}]`,
```

Comment:
This is good, but maybe better if we drop the array.join here and have just a single literal?